### DISCUSSION TITLE
General Talk

### DISCUSSION THREAD

**Discussion by FrutyX** [DinoDevs/GladiatusCrazyAddon#390] [General]
This discussion is for the general talk (no specified topics).

Please do respect the [Code of conduct](https://github.com/DinoDevs/GladiatusCrazyAddon/blob/b7435eb4dafa13a8ac090aca86c14b9ab9c1ef93/.github/CODE_OF_CONDUCT.md).

**Comment by SleepingShadow-bit** [comment]
Is your merge done?
My server was merged yesterday and its really nice about auctions house, more acc and so =)

**Reply by FrutyX** [reply]
Merge experience? EN51? Yes, my experience is "Website not available", constant crashing, timing out, the server is having a really bad time to keep up on 5x speed. Today, it seems its a little bit better, but that's because right know it's early and not that many players are online right now.

**Reply by FrutyX** [reply]
Actually what do you mean by auctions house? I don't see any difference, nor any changelog?

**Reply by SleepingShadow-bit** [reply]
Oh shit u are on en51... I also read the "news" about them.

No i mean the auction Action in the AH =) overbid, overbid...

**Comment by FrutyX** [comment]
With all that is happening with those merges, it really gives me the "Thanos mood", close the tab and never look back....

**Comment by GramThanos** [comment]
Haha, I just wanted to share this printscreen :P
![image](https://user-images.githubusercontent.com/14858959/182569942-e805cfa7-3aae-463c-a7ae-1a241c4f2c97.png)

**Comment by FrutyX** [comment]
![experience](https://user-images.githubusercontent.com/23278552/187043721-b4fd078d-004c-4bbd-a763-ce736886800c.PNG)

Gladiatus server experience :D (GCA disabled).

**Comment by FrutyX** [comment]
How is Apo doing? Haven't seen him in a while, hope he is good.

**Reply by GramThanos** [reply]
I bet he is drinking mojito somewhere...

**Reply by GramThanos** [reply]
probably with @MrGnDkS

**Reply by FrutyX** [reply]
> I bet he is drinking mojito somewhere...

Plot twist: He was actually with me 
![plot](https://user-images.githubusercontent.com/23278552/187091343-bcec8e73-05cc-41f2-8443-9636e036965a.jpg)

**Reply by GramThanos** [reply]
Double plot twist he is with me.

**Reply by FrutyX** [reply]
![shocked-face-shock](https://user-images.githubusercontent.com/23278552/187091535-1bc781ac-8b1a-4756-8f1d-a9ae53da8e88.gif)

**Reply by GreatApo** [reply]
I am gooood, thanks for asking. I just don't have access to my laptop lately.

**Comment by FrutyX** [comment]
Do you know something about this? Maybe it's an old thing I am not aware about, but I have no clue why there is a material on some player profiles, I have noticed this long time ago before the merges, and all extensions are disabled.

![ok](https://user-images.githubusercontent.com/23278552/187280306-d4135933-d4b2-40d1-9899-a82e0eb37ff5.PNG)

**Reply by GramThanos** [reply]
I have no idea, but I would love to check it out. Can you share the link?

**Reply by FrutyX** [reply]
https://s51-en.gladiatus.gameforge.com/game/index.php?mod=player&p=444215

**Reply by GramThanos** [reply]
It is full of nulls and errors... I want to recreate it!

**Reply by FrutyX** [reply]
Ahahah, let me know how that goes, it's an interesting thing, probably some bugs in their codes, not sure. There are more profiles like this, but I don't have exactly a name list, just found this one randomly.

**Comment by GramThanos** [comment]
Coding comment/tips:

- don't mix spaces with tabs indentation (we use tabs so that you can specify tabs width on your editor based on your preferences)
- it is better to use `if (! condition) return;` rather than multiple nested `if`s
- avoid using `var`, `let` and `const` will most probably cover all your needs.
- it is good to leave the last line of the file empty

@FrutyX @GreatApo

**Reply by FrutyX** [reply]
Yes boss, please don't hurt me boss. You know I am a magician, not a programmer 😄 

Also, noted.

**Comment by Ondraczz3** [comment]
I don't believe the add-on will still work.

**Reply by GramThanos** [reply]
... well... history has shown that this thing just refuses to die.

**Reply by FrutyX** [reply]
He he he...

**Reply by Ondraczz3** [reply]
So far, with the addon I can only sell on the market, no shipments or anything else.

**Reply by Ondraczz3** [reply]
Item sold on the marketFri, 18.10.2024 - 00:23:20
You sold the item "Cheese Roll" on the market and get 5,000,000

Item sold on the marketMon, 06.01.2025 - 22:50:56
You sold the item "Gold Ore" on the market and get 100,000

Item sold on the marketMon, 06.01.2025 - 22:08:52
You sold the item "Copper" on the market and get 1,000,000

Item sold on the marketMon, 06.01.2025 - 22:08:50
You sold the item "Country Mark" on the market and you get 1,000,000
The item was sold on the marketThu, 06.03.2025 - 22:03:22
You sold the item "Rynight's Healing Potion of Trust" on the market and you get 50,000

The item was sold on the marketThu, 06.03.2025 - 22:03:22
You sold the item "Quint's Healing Potion of Hatred" on the market and you get 50,000

The item was sold on the marketThu, 06.03.2025 - 22:03:21
You sold the item "Marco's Healing Potion of Speed" on the market and you get 50,000

The item was sold on the marketThu, 06.03.2025 - 22:03:21
You sold the item "Stoybaer's Healing Potion of Sacrifice" on the market and received 50,000

**Reply by FrutyX** [reply]
@Ondraczz3 For now, you either disable the addon completely or disable those features that are bugged.

[This](https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/PROGRESS.md) is the progress list for our Beta Dev version that is not made public yet, in the works.

**Reply by Ondraczz3** [reply]
The Ark of God's Destiny doesn't work either.

**Reply by FrutyX** [reply]
I take it you mean the Chest? That was patched too, it just wasn't added on the progress list yet...

**Reply by FrutyX** [reply]
> ... well... history has shown that this thing just refuses to die.

It is your child, but also your curse.
